.github/workflows/build.yml

```diff
@@ -118,3 +118,20 @@ jobs:
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           RUNTIME_PARAM: ${{ matrix.runtime-param }}
         run: ./scripts/run_integration_tests.sh
+
+      - name: Send success metric
+        env:
+          DD_API_KEY_SERVERLESS: ${{ secrets.DD_API_KEY_SERVERLESS }}
+        run: ./scripts/send_status_metric.sh 0 $DD_API_KEY_SERVERLESS
+
+  run-if-failed:
```

Comment:
I would name this "integration-test-failure" so we know that it runs when the integration tests failed, not the other tests/linter/etc.

Alternatively, can we make this a step inside the integration tests job instead of a separate job? This doc makes me think it might be possible: https://docs.github.com/en/actions/learn-github-actions/contexts#job-status-check-functions

Reply:
Specifically, I am looking at `steps.<step id>.outcome` and `continue-on-error`

Reply:
Renamed! I think I preferred a separated step as it shows up nicely in the workflow diagram (screenshot in the description)

Reply:
> This looks good, but I thought we were going to add this metric for the extension integration tests, not the library integration tests. Since these tests already run on every PR, is there a risk that we wouldn't notice a failure?

Yes I think so, since those tests are also run every day, we might have flaky tests which may have passed during the PR but will fail the day after. What do you think?